### PR TITLE
Add v1.7 release note

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  - Add v1.7 release note with doc links

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by CharlieSYH — APPROVED** [review]
Made one adjustment.
image
Links are all checked. Good to merge.

**Comment by CharlieSYH** [comment]
And here
image

**Comment by CharlieSYH** [comment]
And here. Please verify the command.
image

File: src/pages/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,60 @@ slug: /release-notes
...
+- Supports adaptive scaling for streaming jobs by default. [#14873](https://github.com/risingwavelabs/risingwave/pull/14873). See [Cluster scaling](/docs/current/k8s-cluster-scaling/).
+
+### Assets
+
+- Run this version from Docker:<br/>
+    `docker run -it --pull=always -p 4566:4566 -p 5691:5691 risingwavelabs/risingwave:v1.7.0 risingwave`
+- [Prebuilt all-in-one library for Linux](https://github.com/risingwavelabs/risingwave/releases/download/v1.7.0/risingwave-v1.7.0-x86_64-unknown-linux-all-in-one.tar.gz)
+- [Source code (zip)](https://github.com/risingwavelabs/risingwave/archive/refs/tags/v1.7.0.zip)
+- [Source code (tar.gz)](https://github.com/risingwavelabs/risingwave/archive/refs/tags/v1.7.0.tar.gz)
+- [risectl-v1.7.0-x86_64-unknown-linux.tar.gz](https://github.com/risingwavelabs/risingwave/releases/download/v1.7.0/risectl-v1.7.0-x86_64-unknown-linux.tar.gz)
```
Comment (hengm3467) [inline src/pages/release-notes.md]: @huangjw806 @lmatz Do you happen to know why we have this `risectl` file here? How is it normally used? If it is useful, we can add some description. If not, we can remove it here.

**Review by hengm3467 — APPROVED** [review]

File: src/pages/release-notes.md [hunk 2]
```diff
@@ -9,6 +9,60 @@ slug: /release-notes
...
+- Updates StarRocks sink connector parameters. [#14823](https://github.com/risingwavelabs/risingwave/pull/14823). See [Sink data from RisingWave to StarRocks](/docs/current/sink-to-starrocks/).
+- Introduces `snapshot` option to allow users to disable CDC backfill and to only consume from the latest changelog. [#14718](https://github.com/risingwavelabs/risingwave/pull/14718). See [Ingest data from PostgreSQL CDC](/docs/current/ingest-from-postgres-cdc/) and [Ingest data from MySQL CDC](/docs/current/ingest-from-mysql-cdc/).
+- Sets the default value of `transactional` parameter to `true` for MySQL and Postgres CDC shared sources. [#14899](https://github.com/risingwavelabs/risingwave/pull/14899). See [Ingest data from PostgreSQL CDC](/docs/current/ingest-from-postgres-cdc/) and [Ingest data from MySQL CDC](/docs/current/ingest-from-mysql-cdc/).
+
+#### Installation and deployment
+
+- [Pre-Release] Supports the standalone mode to run RisingWave in a single process. [#14951](https://github.com/risingwavelabs/risingwave/pull/14951). See [Quick start](/docs/current/get-started/).
```
Comment (ShanlanLi) [inline src/pages/release-notes.md:47]:
```suggestion
- [Pre-Release] Supports the standalone mode to run RisingWave in a single process. [#14951](https://github.com/risingwavelabs/risingwave/pull/14951). See [Quick start](/docs/current/get-started/).
- Supports Alibaba Cloud OSS as the storage backend. See [Start RisingWave using Docker Compose](/docs/current/risingwave-docker-compose/).
```

**Review by ShanlanLi — APPROVED** [review]
LGTM, add Alibaba cloud storage to installation and deployment.

File: src/pages/release-notes.md (hunk 1 above)
Comment (huangjw806) [reply src/pages/release-notes.md]: risectl is the command line tool for managing risingwave.
The purpose of release is to allow users to directly obtain this tool without having to compile it themselves.

**Review by huangjw806 — APPROVED** [review]
LGTM!

File: src/pages/release-notes.md (hunk 1 above)
Comment (CharlieSYH) [reply src/pages/release-notes.md]: image

**Pull request merged**